lib/tizen_sdk.dart

```diff
@@ -253,25 +253,27 @@ class TizenSdk {
     String workingDirectory, {
     String? buildType,
     String? signingProfile,
-  }) {
-    _processUtils.run(
+  }) async {
+    final RunResult result = await _processUtils.run(
       <String>[
         tzCli.path,
         'set',
         if (buildType != null) ...<String>['-b', buildType],
         if (signingProfile != null) ...<String>['-s', signingProfile],
-        '-w',
-        workingDirectory,
       ],
+      workingDirectory: workingDirectory,
     );
 
+    if (result.exitCode != 0) {
+      throwToolExit('Failed to build .NET application:\n$result');
```

Comment:
The return type of this method is `RunResult`. A better practice would be to just return the `result` here.